report/outbreak_no_intervention.qmd

```diff
@@ -0,0 +1,136 @@
+---
+title: H5N1 outbreak scenarios
+---
+
+This report requires two R packages: {epichains} and {ggplot2}, the former to simulate the branching process model and the latter to visualise the results.
+
+```{r}
+# install from outbreak_dist branch of {epichains}
+# remotes::install_github("epiverse-trace/epichains@outbreak_dist")
+library(epichains)
+library(ggplot2)
+```
+
+The `simulate_scenarios()` function from {epichains} calculates the proportion of outbreak statistic across values of the reproduction number ($R$).
+
+Here we run the function for both Poisson (`"rpois"`) and Negative Binomial (`"rnbinom"`) distributions, calculating both the size and length of the outbreak. Size is the total number of cases produced by a chain before it goes extinct. Length is the total number of generations reached by a chain before it goes extinct. See `?epichains::simulate_chain_stats` for more information on the branching process simulation.
+
+We simulate using a reproduction number ($R$) between 0.1 and 0.9 at 0.1 intervals, and for the Negative Binomial offspring distribution we vary the dispersion parameter ($k$) between 0.5 and 2.5 at 0.5 intervals. Smaller values of $k$ signify more overdispersion in the transmission spreading, in other words transmission is more heterogeneous and superspreding events are more likely. As $k$ increases individual-level transmission is more homogeneous and when $k \rightarrow \infty$ the Negative Binomial equals the Poisson offspring distribution. 
+
+```{r}
+df <- simulate_scenarios(
+  offspring_dist = c("rpois", "rnbinom"), 
+  statistic = c("size", "length"), 
+  R_seq = seq(0, 0.9, 0.1), 
+  k_seq = seq(0.5, 2.5, 0.5), 
+  breaks = c(0, 2, 5, 10, 20, 50, 100, Inf)
+)
+```
+
+The function produces a `data.frame` that contains the outbreak size or length in intervals (`$interval`), the proportion of outbreaks that fall into that interval (`$Freq`), the reproduction number (`$R`), the dispersion parameter (`$k`, for Poisson distributions $k$ is set to `NA`), the offspring distribution (`$offspring_dist`), and the outbreak statistic (`$statistic`, either size or length). 
```

Comment:
Would suggest to cut as well.

```suggestion
The function produces a `data.frame` that contains the outbreak size or length in intervals (`$interval`), the proportion of outbreaks that fall into that interval (`$Freq`), the reproduction number (`$R`), the dispersion parameter (`$k`, for Poisson distributions $k$ is set to `NA`), the offspring distribution (`$offspring_dist`), and the outbreak statistic (`$statistic`, either size or length). 
```

Reply:
I'm going to leave this in for now as the report is still WIP. We can easily remove it if we agree it can go in the future.